examples/js/loaders/GLTFLoader.js

```diff
@@ -1547,59 +1528,74 @@ THREE.GLTFLoader = ( function () {
 
 	}
 
-	function isArrayEqual( a, b ) {
+	function createGeometryKey( primitiveDef ) {
```

Comment:
Note that this method is being called below with a `BufferGeometry` as an argument, which it doesn't support. For clarity the method could be renamed as `createPrimitiveKey`.